hecuba_py/src/hdict.py

```diff
@@ -493,7 +525,12 @@ def make_persistent(self, name):
                     log.error("Error creating the Qbeast trigger: %s %s", trigger_query, ex)
                     raise ex
 
-        persistent_columns = [{"name": tup[0], "type": tup[1]} for tup in persistent_values]
+        # persistent_columns = [{"name": tup[0], "type": tup[1]} for tup in persistent_values]
+        persistent_columns = []
+        if not self._has_embedded_set:
```

Comment:
I don't like having all these `if not self._has_embedded_set`
For now is OK, but we need to rethink how we organize the class